layers/+readers/djvu/funcs.el

```diff
@@ -0,0 +1,71 @@
+(defun djvu-toggle-semi-continuous-scrolling ()
+  (interactive)
+  (setq djvu-semi-continuous-scrolling (if djvu-semi-continuous-scrolling
+                                       nil
+                                     t))
+  (message "Djvu alternative scrolling %s" (if djvu-semi-continuous-scrolling
+                                               "enabled"
+                                             "disabled")))
+
+(defun spacemacs/djvu-advise-image-toggle (orig-func &rest args)
+  (djvu-image-toggle))
+
+;; djvu-continuous of djvu.el does not work with djvu3.el
+(defun spacemacs/djvu-scroll-up-or-next-page ()
+  (interactive)
+  (if (not djvu-semi-continuous-scrolling)
+      (if djvu-image-mode
+          (djvu-image-scroll-up)
+        (evil-next-visual-line))
+    (scroll-up-line 5)
+    (when (= (window-vscroll) 0)
+      (djvu-next-page 1))))
+
+(defun spacemacs/djvu-scroll-down-or-previous-page ()
+  (interactive)
+  (if (not djvu-semi-continuous-scrolling)
+      (if djvu-image-mode
+          (djvu-image-scroll-down)
+        (evil-previous-visual-line))
+    (if (not (= (window-vscroll) 0))
+        (scroll-down-line 5)
+      (djvu-prev-page 1)
+      (scroll-up-command))))
+
+(defun spacemacs/djvu-fast-search (regexp)
+  (interactive "sSearch (regexp): ")
+  (when djvu-image-mode
+    (djvu-image-toggle))
+  (spacemacs/djvu-search-forward regexp))
+
+(defun spacemacs/djvu-search-forward (query)
+  "Search forward for match for REGEXP.
+
```

Comment:
No need to add a blank line after first line of docstring.
It's added automatically.